portfolio/src/main/webapp/authenticationScript.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Shows different messages according to user log in status.
+ */
+function addLoginOrLogoutMessages() {
+  const anchor = document.createElement('a');
+  const initialMessage = document.createElement('span');
+  const linkMessage = document.createElement('span');
+  const container = document.getElementById('login-container');
+  fetch('/login').then(response => response.json()).then((loginData) => {
+    if (loginData.logStatus) {
+      initialMessage.innerText = `Welcome, ${loginData.userEmail}!`;
+      anchor.innerText = `Log out.`
+      anchor.href = `${loginData.loginUrl}`;
+    } else {
+      initialMessage.innerText = 'Please log in to leave a comment';
+      anchor.innerText = `Log in.`
+      anchor.href = `${loginData.loginUrl}`;
+      const form = document.getElementById('form-container');
+      form.style.display = 'none';
+    }
+    linkMessage.appendChild(anchor);
+    container.appendChild(initialMessage);
+    container.appendChild(linkMessage);
+    if (loginData.logStatus) {
+      addNicknameLink(container);
+    }
+  });
+}
+
+/**
+  * If the user is logged in, they can change or set up their nickname. 
+  * So they can be redirected to the page for nickname set up.
+  */
+function addNicknameLink(container) {
+  const br = document.createElement('br');
```

Comment:
It's generally considered poor form to introduce br's as a way of styling spacing.

Prefer using css for that.

Reply:
Done